environments/staging/app-processes.yml

```diff
@@ -145,6 +145,10 @@ pillows:
       gevent_workers: 4
       processor_chunk_size: 1
       dedicated_migration_process: True
+    demo-case-pillow:
+      # add include_demo_topic: True to case-pillow if you remove this to drain the demo topic
```

Comment:
I think this will need some more code in HQ to work the way this suggests.

Reply:
See https://github.com/dimagi/commcare-cloud/pull/6684/commits/46e27426c01f8e7ec62577583580962deec2db4c for an example.